plugins/wasmedge_llmc/llmc_fwd.h

```diff
@@ -13,7 +13,7 @@ struct DataLoader;
 
 GPT2 *gpt2_create(const char *checkpoint_path);
 
-void gpt2_free(GPT2 *model);
+void gpt2_destroy(GPT2 *model);
```

Comment:
Is the renaming required?

Reply:
it's not renaming, we're using the wrong api

Reply:
So that definition is external, I see.